packages/tables/src/Testing/TestsColumns.php

```diff
@@ -429,4 +429,13 @@ public function searchTable(): Closure
             return $this;
         };
     }
+
+    public function searchTableColumn(): Closure
+    {
+        return function (string $column, ?string $search = null): static {
+            $this->set('tableColumnSearchQueries', [$column => $search]);
```

Comment:
Thanks for the PR, @joaoprado!

Maybe we should handle searching multiple columns as well?

Reply:
That is a good idea, for multiple columns I'm thinking the first parameter should be an array and be passed as the `$this->set('tableColumnSearchQueries', $column)` parameter.

Do I need to run any validation with this approach? Like validating the array structure

Reply:
We might rename this to be `searchTableColumns` instead and accept an array with keys being the column and value being the search string? No validation needed.

Reply:
With that approach we are assuming the user is trying to search two columns with the same value and, at least in the use cases I've seen for individual column searching, that seems to not be the intention. It is used most of the time to search records with different values for each column.

I have added a new commit with the approach I mentioned and a [test case](https://github.com/filamentphp/filament/pull/5794/files#diff-4d5f6108cd32b292ec5aa83a683d89a2127590cabb4c97681c5e89f87a6eec88R88) to exemplify this.

Reply:
No, passing an array allows searching multiple values:
```php
searchTableColumns([
    'name' => 'John',
    'city' => 'London',
])
```

Reply:
Could you update this PR to that?

Reply:
Oh ok, I missunderstood what you said before. So we will always want an array of keys and values even if it is just one column. I will send a new commit